Formula/n/netatalk.rb

```diff
@@ -0,0 +1,56 @@
+class Netatalk < Formula
+  desc "File server for Macs, compliant with Apple Filing Protocol (AFP)"
+  homepage "https://netatalk.io"
+  url "https://github.com/Netatalk/netatalk/releases/download/netatalk-4-0-8/netatalk-4.0.8.tar.xz"
+  sha256 "d09d591521b27b91b1c2a6255a2e059af8c9fda43570f983f0a145706e5f7628"
+  license all_of: [
+    "GPL-2.0-only",
+    "GPL-2.0-or-later",
+    "LGPL-2.0-only",
+    "LGPL-2.1-or-later",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "MIT",
+  ]
+  head "https://github.com/Netatalk/netatalk.git", branch: "main"
+
+  depends_on "docbook-xsl" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+
+  depends_on "berkeley-db"           # macOS bdb library is outdated
+  depends_on "libevent"
+  depends_on "libgcrypt"
+  depends_on "mysql"
+  depends_on "openldap"              # macOS LDAP.Framework is deprecated
+
+  uses_from_macos "libxslt" => :build
+  uses_from_macos "perl"
+
+  def install
+    prefix_path = ENV["HOMEBREW_PREFIX"]
+
+    args = %w[
+      -Dwith-afpstats=false
+      -Dwith-init-style=none
+      -Dwith-install-hooks=false
+      -Dwith-spotlight=false
+    ]
+
+    args << "-Dwith-bdb-path=#{prefix_path}/opt/berkeley-db"
+    args << "-Dwith-docbook-path=#{prefix_path}/opt/docbook-xsl/docbook-xsl"
+    args << "-Dwith-pam-config-path=#{prefix_path}/etc/pam.d"
+
+    args << "-Dwith-appletalk=true" if OS.linux?
```

Comment:
Seems weird to set `with-appletalk` on Linux but not macOS. Should this be `if OS.mac?`

Reply:
Curiously, Linux and NetBSD are the only two OSes that still support the AppleTalk (DDP) transport layer in kernel. 

Apple removed support from the Darwin kernel some time around 10.9 Mavericks. So on macOS we are limited to only the TCP/IP transport layer for Netatalk.

Reply:
I see, in which case:
```suggestion
    args << "-Dwith-appletalk=#{OS.linux?}"
```
You should probably just create all these `args` as a single array instead of appending arguments to it one at a time.

Reply:
> I see, in which case:

This is definitely a more elegant solution. Will adopt.

> You should probably just create all these args as a single array instead of appending arguments to it one at a time.

Fair enough. I was aiming to keep the `%w` literal for that clean list syntax, but the strings that needed interpolation kept adding up. I'll make the change.